src/modules/vendor/pages/ni/config/ActivityProfile.vue

```diff
@@ -60,6 +92,39 @@ export default {
       console.error(e);
     }
   },
+  methods: {
+    async refreshActivity () {
+      try {
+        this.activity = await Activities.getById(this.activityId);
+      } catch (e) {
+        console.error(e);
```

Comment:
il faut que tu mettes `this.activities = [];` dans le catch pour que ce soit toujours bien defini

Reply:
done